frontend/components/Landing/CountDown.vue

```diff
@@ -0,0 +1,62 @@
+<script setup>
+const props = defineProps({
+  date: String,
+})
+
+let finish = false;
+let isToday = false;
+let isTomorrow = false;
+
+let now = new Date();
+let endDate = new Date(props.date);
+
+if (endDate.getDate()  == now.getDate()  && 
+    endDate.getMonth() == now.getMonth() &&
+    endDate.getYear()  == now.getYear()) {
+    isToday = true;
+} else if (
```

Comment:
```suggestion
if (endDate.toDateString() === now.toDateString()) {
    isToday = true;
} else if (
```
Maybe that its more readable than three conditional if statement?

Reply:
this is much better :) i will change it